docs/common/DocsExample.vue

```diff
@@ -1,7 +1,17 @@
 <template>
 
   <div class="docs-example">
-    <div class="code-toggle-button">
+    <div
+      class="code-button"
+      :style="codeButtonStyle"
+    >
+      <KIconButton
+        v-if="loadExample"
+        appearance="raised-button"
+        icon="github"
+        :tooltip="githubTooltip"
+        @click="redirectToGitHub"
```

Comment:
We could also add an `ariaLabel` here to make it a little bit more clear, and repurpose the tooltip string for that.

Reply:
Got it! Will update!